app/editor/src/components/formik/checkbox/FormikStringEnumCheckbox.tsx

```diff
@@ -0,0 +1,64 @@
+import { getIn, useFormikContext } from 'formik';
+
+import { FormikCheckbox, IFormikCheckboxProps } from '.';
+
+export interface IFormikStringEnumCheckboxProps<TEnum> extends IFormikCheckboxProps {
+  /** values must be numeric to work with bitwise logic. */
+  value: any | TEnum | TEnum[];
+  deliminator?: string;
+}
+
+/**
+ * FormikStringEnumCheckbox provides a way to handle a string/enum array of values.
+ * @param param0 Component properties.
+ * @returns A FormikStringEnumCheckbox component.
+ */
+export const FormikStringEnumCheckbox = <TEnum extends unknown>({
```

Comment:
The .NET api is configured to return enum values as strings instead of numbers.  This handles an array of string values.